app/controllers/application_controller/filter.rb

```diff
@@ -1,10 +1,13 @@
 # Filter/search/expression methods included in application.rb
+require_relative '../../helpers/miq_expression/filter_subst'
```

Comment:
Are these methods UI specific though, or are they related to any MiqExpression?  Can they just be a part of the MiqExpression class (even as class methods)?